src/codegen/function.rs

```diff
@@ -14,10 +14,10 @@ use writer::ToCode;
 
 pub fn generate(w: &mut Write, env: &Env, analysis: &analysis::functions::Info,
     in_trait: bool, only_declaration: bool, indent: usize) -> Result<()> {
-
     let mut commented = false;
     let mut comment_prefix = "";
     let mut pub_prefix = if in_trait { "" } else { "pub " };
+    try!(writeln!(w, ""));
```

Comment:
Without an `if` this has an additional effect in trait definitions. Is that inteded? It doesn't match the commit/PR note.